agent/tool-scripts/meta.json

```diff
@@ -0,0 +1,48 @@
+{
+	"transient":{
+		"blktrace": null,
+		"bpftrace": null,
+		"cpuacct": null,
+		"disk": null,
+		"dm-cache": null,
+		"docker": null,
+		"docker-info": null,
+		"external-data-source": null,
+		"haproxy-ocp": null,
+		"iostat": null,
+		"jmap": null,
+		"jstack": null,
+		"kvm-spinlock": null,
+		"kvmstat": null,
+		"kvmtrace": null,
+		"lockstat": null,
+		"mpstat": null,
+		"numastat": null,
+		"oc": null,
+		"openvswitch": null,
+		"pcp": null,
+		"perf": null,
+		"pidstat": null,
+		"pprof": null,
+		"proc-interrupts": null,
+		"proc-sched_debug": null,
+		"proc-vmstat": null,
+		"prometheus-metrics": null,
+		"qemu-migrate": null,
+		"rabbit": null,
+		"sar": null,
+		"strace": null,
+		"sysfs": null,
+		"systemtap": null,
+		"tcpdump": null,
+		"turbostat": null,
+		"user-tool": null,
+		"virsh-migrate": null,
+		"vmstat": null
+	},
+
+	"persistent":{
+		"node-exporter": "prometheus",
```

Comment:
We could work around this later by inspecting the type, but to allow for future expansion, maybe starting with a record would be more flexible? E.g., `"node-exporter": {"source": "prometheus"}` ???